src/account/templates/add-login.html

```diff
@@ -2,6 +2,10 @@
 
 {% load crispy_forms_tags %}
 
+{% block stylesheets %}
+<link rel="stylesheet" type="text/css" href="/static/lib/datahub-font/css/MIT-Icons.css" />
+{% endblock %}
```

Comment:
I've been trying to have endblocks denote what they're ending, since that makes template more legible later on.